addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechsJsonParser.java

```diff
@@ -404,6 +412,10 @@ private List<Map<String, Map<String, Map<String, AppPattern>>>> jsonToAppPattern
                     }
                 }
             }
+        } else if (json instanceof JSONArray) {
+            LOGGER.debug(
+                    "Ignoring {} DOM Pattern JSONArray, it will be added as Simple DOM Patterns",
+                    app.getName());
```

Comment:
A ton of stuff was hitting the condition below, and it took me a while to realize what was going on. I'd even implemented some additional handling before I realized it was just done after the fact 🤦‍♂️

Reply:
Done and done